features/ec2/step_definitions/ec2.js

```diff
@@ -1,19 +1,8 @@
 const { Before, Given, Then } = require("@cucumber/cucumber");
-
-const waitForVolumeAvailableCallback = (ec2, volumeId, callback) => {
```

Comment:
I just realized these were written before waiters were available in the SDK. Thanks for cleaning them up!